client/components/vertical-nav/index.jsx

```diff
@@ -13,4 +19,4 @@ const VerticalNav = React.createClass( {
 	}
 } );
 
-export default VerticalNav;
+export default withStyles( style )( VerticalNav );
```

Comment:
Could we omit `import style from './style';` line and use the following instead:

```js
export default withStyles( './style' )( VerticalNav );
```
:) ?